stripe/src/main/java/com/stripe/android/CustomerSession.java

```diff
@@ -226,6 +354,54 @@ private Calendar getCalendarInstance() {
         return mProxyNowCalendar == null ? Calendar.getInstance() : mProxyNowCalendar;
     }
 
+    static Source addCustomerSourceWithKey(
+            @NonNull EphemeralKey key,
+            @NonNull String sourceId,
+            @Nullable StripeApiProxy proxy) {
+        Source source = null;
+        try {
+            if (proxy != null) {
+                return proxy.addCustomerSourceWithKey(
+                        key.getCustomerId(),
+                        sourceId,
+                        key.getSecret());
+            }
+            source = StripeApiHandler.addCustomerSource(
+                    key.getCustomerId(),
+                    sourceId,
+                    key.getSecret());
+        } catch (InvalidRequestException invalidException) {
+            // Then the key is invalid
```

Comment:
Just curious, why do we log the StripeException but not the InvalidRequestException?